doc/source/whatsnew/v0.20.0.txt

```diff
@@ -362,3 +362,4 @@ Bug Fixes
 
 
 - Bug in ``pd.read_csv()`` for the C engine where ``usecols`` were being indexed incorrectly with ``parse_dates`` (:issue:`14792`)
+- Bug in dtypes returned by comparison methods (e.g., ``lt``, ``gt``, ...) against a constant for an empty DataFrame (:issue:`15077`)
```

Comment:
Ok, thanks for reviewing it. I thought it was enough to link the issue for more details.